### PR TITLE
Add V2 IPFS metadata support for decisions

### LINKED ISSUE #3887: Decisions:Add support for V2 ipfs metadata
## Description

Update Simple Decisions to use V2 ipfs metadata (@colony/colony-event-metadata-parser) when accessing IPFS.

Please note, this will be blocked until #3681 is rebased & merged into Master, and then Simple Decision Feature branch is rebased with Master.

### PR DESCRIPTION
**Pull request by chinins**
## Description

This PR adds V2 IPFS metadata support fro decisions. You need to run npm i to have an updated version of the package.

resolves #3887

### CODE REVIEW

**Review by danbr — APPROVED** [review]
Code looks good. Decision details are being retrieved (so must've been uploaded successfully) - well done.
There is a (an existing) bug that is stopping me from checking the objection details.

**Review by danbr — CHANGES_REQUESTED** [review]
I noticed something out of the 'new' way of accessing metadata. We should be using getAnnotationMsgFromResponse rather than calling JSON.parse directly.

**Review by danbr — APPROVED** [review]
I found an improvement that corrects a bug, but also good practice to use the new metadata parsing lib. Can you please replace.......
I'll approve. Well done.

File: src/modules/dashboard/components/ActionsPage/ActionPageDecisionWithIPFS/ActionPageDecisionWithIPFS.tsx [hunk 1]
```diff
@@ -72,8 +73,9 @@ const ActionPageDecisionWithIPFS = ({
       } as DecisionDetails;
     }
 
-    // @TODO - add V2 IPFS support inc validation
-    const details: DecisionDetails = JSON.parse(ipfsDataJSON);
+    const details: DecisionDetails = getDecisionDetailsFromResponse(
```
[the remark this answers is not in the document]
Comment (chinins) [reply src/modules/dashboard/components/ActionsPage/ActionPageDecisionWithIPFS/ActionPageDecisionWithIPFS.tsx:79]: All done!

**Review by jakubcolony — APPROVED** [review]
Looks good to me as far as I can tell

**Pull request merged**